src/main/java/de/tum/in/www1/artemis/domain/Submission.java

```diff
@@ -69,6 +69,13 @@
     @Column(name = "submission_date")
     private ZonedDateTime submissionDate;
 
+    public Submission() {
```

Comment:
Please remove these constructors. We don’t need them and it’s unclear if they influence some behavior in hibernate or Jackson

Reply:
have been removed